### PR TITLE
[Merged by Bors] - consolidated examples

### PR DESCRIPTION
**Pull request by maltesander**
## Description

fixes https://github.com/stackabletech/druid-operator/issues/156

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: examples/simple-druid-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (fhennig) [inline examples/simple-druid-cluster.yaml:1]: I just skimmed the file, but what's the difference to the `derby` variant? If they are the same, i don' think we should keep both.

**Review by fhennig — COMMENTED** [review]
I don't know about the structure. Now that there are only single files (and the readmes) in the directories I think it would be better to have all files in the same directory. Maybe the README content could be put inside a comment into the yaml but maybe it's too hidden then. Or we have just a single readme with sections of instructions per different yaml file.

File: examples/simple-druid-cluster.yaml (hunk 1 above)
Comment (maltesander) [reply examples/simple-druid-cluster.yaml:1]: Nothing, didnt want to destroy your structure!

**Comment by maltesander** [comment]
> I don't know about the structure. Now that there are only single files (and the readmes) in the directories I think it would be better to have all files in the same directory. Maybe the README content could be put inside a comment into the yaml but maybe it's too hidden then. Or we have just a single readme with sections of instructions per different yaml file.

I actually like the folder approach with one readme per folder. Especially if it gets more complicated i dont want to maintain one big readme. I can remove the derby example since its the same as the one in the root but would keep the rest as is.

**Review by fhennig — APPROVED** [review]
LGTM

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**